### ISSUE TITLE
Multiple keys?

### ISSUE THREAD

**Issue by castillar** [kryptco/kr#92]
I prefer using ed25519 keys wherever possible since that seems to be the future direction of things, but I still have a lot of boxes running older distributions (RHEL/CentOS 6, e.g.) that only support RSA. Would it be possible to support more than one key per user (perhaps requiring different 'key comments' for each)?

**Comment by kcking** [comment]
I totally agree -- you can follow multi-key support for [iOS](https://github.com/kryptco/kryptonite-ios/issues/44) or [android](https://github.com/kryptco/kryptonite-android/issues/38)

**Comment by castillar** [comment]
Woohoo! I'll keep an eye on it, thanks!
